README.md

```diff
@@ -75,14 +75,14 @@ Add the following HTML:
   <div class="bs-stepper-header" role="tablist">
     <!-- your steps here -->
     <div class="step" data-target="#logins-part">
-      <button type="button" class="btn btn-link step-trigger" role="tab" aria-controls="logins-part" id="logins-part-trigger">
+      <button type="button" class="btn btn-link bs-stepper-btn step-trigger" role="tab" aria-controls="logins-part" id="logins-part-trigger">
```

Comment:
we shouldn't use both `.step-trigger` and `.bs-stepper-btn` but we should be able to use those two classes